tools/python/boutiques/localExec.py

```diff
@@ -34,46 +34,45 @@ def __init__(self,desc,options={}):
     # The set of output parameters from the json descriptor
     self.outputs     = self.desc_dict['output-files'] # Structure: [ {id:...},...,{id:...} ]
     # The set of parameter groups, according to the json descriptor
-    self.groups     = self.desc_dict['groups'] if 'groups' in self.desc_dict.keys() else []
+    self.groups     = self.desc_dict['groups'] if 'groups' in list(self.desc_dict.keys()) else []
```

Comment:
These were added in the 2to3 conversion, so that it reliably works consistently with both. Getting keys in Python3 gives you a `dict_keys` object, not a list.